app/install/aws-linux.md

```diff
@@ -54,7 +54,7 @@ baseurl=https://kong.bintray.com/kong-rpm/amazonlinux/amazonlinux
     $ sudo yum install -y kong
     ```
 
-2. **Prepare your database**
+2. **Prepare your database or declarative configuration file**
```

Comment:
```suggestion
2. **Prepare your database or declarative configuration file**

Kong can run either with or without a database:

* When using a database, you will use the `kong.conf` configuration file for setting Kong's configuration properties at start-up and the database as storage of all configured entities, such as the routes and services to which Kong proxies.
* When not using a database, you will use `kong.conf` its configuration properties and a `kong.yml` file for specifying the entities as a declarative configuration.

**Using a database**

```

Reply:
^ similar for "Prepare your database..." entries in all files

Reply:
I have added this intro, the **Without a database** subtitle, and a section containing `kong config init` inside that on all the pages.